docs/program/_index.md

```diff
@@ -3,8 +3,278 @@ title: "Program a Robot"
 linkTitle: "Program Robots"
 childTitleEndOverwrite: "Program Robots"
 description: "Use the SDK of your preferred language to write code to control your robots."
-weight: 50
-simple_list: false
+weight: 45
+no_list: true
 type: docs
 images: ["/img/code.png"]
+aliases:
+  - "product-overviews/sdk-as-client"
+  - "program/sdk-as-client"
+  - "program/sdks"
 ---
+
+Viam offers software development kits (SDKs) in popular languages which
+
+- Streamline connection, authentication, and encryption against a server using {{< glossary_tooltip term_id="webrtc" >}}
+- Enable you to interface with robots without calling the `viam-server` [gRPC APIs for robot controls](https://github.com/viamrobotics/api) directly
+
+![Diagram showing how a client connects to a robot with Viam. Diagram shows a client as a computer sending commands to a robot. Robot 1 then communicates with other robotic parts over gRPC and WebRTC and communicating that information back to the client.](img/sdks/robot-client.png)
+
+Use the SDK of your preferred language to write code to control your robots.
+
+Viam currently offers SDKs for the following languages:
+
+- [Python SDK](https://python.viam.dev/)
+- [Go SDK](https://pkg.go.dev/go.viam.com/rdk)
+- [TypeScript SDK](https://ts.viam.dev/)
+- [C++ SDK (alpha)](https://cpp.viam.dev/)
+
+## Requirements
+
+Before you get started with your program, ensure that you have [installed and connected to `viam-server`](/installation/) on the computer you want to use to control a robot [(likely a single-board computer)](/components/board/#configuration) and [configured a robot](/manage/configuration/#local-setup).
+
+The easiest way to do this is:
+
+1. Go to [app.viam.com](https://app.viam.com/).
+2. Create a new robot.
+3. Go to the **Setup** tab and follow the instructions there.
+4. [Configure](/manage/configuration/) your robot.
+
+Next, to install your preferred Viam SDK on your development machine, run one of the following commands in your terminal:
```

Comment:
Do we want to here mention that said development machine must be Linux or Mac?
Also, they could run SDKs from their robots (SBCs) themselves, and there are many valid reasons to do so.  I think we should mention this as an option.

Reply:
Great points, done (I think)